lib/App.js

```diff
@@ -1835,6 +1826,15 @@ class App {
         }
       ]);
 
+      const { argumentPlaceholder } = await inquirer.prompt([
+        {
+          type: 'input',
+          name: 'argumentPlaceholder',
+          message: 'Enter the placeholder for the argument',
+          validate: input => input.length > 0,
+        }
+      ]);
+
```

Comment:
text, number, autocomplete, range, date, time and device allow a placeholder.
I believe dropdown, color and droptoken don't.

Reply:
Yes

Reply:
actually range also doesn't have a placeholder!